roles/nf-core-biocontainer-temp/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: install nf-core/rnaseq
+  command: "{{ nf_core_env }}/bin/nf-core -v download {{ item.name }}
+   --compress none --outdir {{ sw_path }}{{ item.name }} --release {{ item.release }}"
+  environment:
+    PATH: "{{ nf_core_env }}/bin:{{ ansible_env.PATH }}"
+  args:
+    chdir: "{{ sw_path }}"
+  with_items: "{{ pipelines }}"
+
+- name: Create directory for rnaseq biocontainers
+  file:
+    path: "{{ ngi_containers }}/biocontainers"
+    state: directory
+    mode: 02775 #ug=rwx, o=rx, g+s
+
+- name: Create temp directories for download
+  file:
+    path: "{{ ngi_containers }}/download"
+    state: directory
+
+#Download these for rnaseq 3 containers until nf-core dowload is updated
+#wget -L https://dl.dropboxusercontent.com/s/5venz3l2wihaqdp/nf-core-rnaseq_v3.0.tar.gz
+#wget -L https://dl.dropboxusercontent.com/s/0w2e72pz2iuj37e/nf-core-rnaseq_v3.0.tar.gz.md5
+
+- name: Download containers
+  get_url:
+    url: https://dl.dropboxusercontent.com/s/5venz3l2wihaqdp/nf-core-rnaseq_v3.0.tar.gz
+    dest: "{{ ngi_containers }}/download/nf-core-rnaseq_v3.0.tar.gz"
+    remote_src: yes
+
+- name: Unarchive downloaded file
+  unarchive:
+    src: "{{ ngi_containers }}/download/nf-core-rnaseq_v3.0.tar.gz"
+    dest: "{{ ngi_containers }}/biocontainers"
```

Comment:
I think it would be nice if the rnaseq containers were put directly in `biocontainers` and not in in subfolder `nf-core-rnaseq_v3.0`. This is under the assumption that other pipeline will share the same containers in the future. But I don't know if it's do-able directly in the unarchive module and hence might require some extra steps. Maybe not worth it. Let me know what you think.

Reply:
Done! I though as the other pipelines would be downloaded through `nf-core download` most probably, it could be better to keep this separate. But you're right, it would be better to start using it the right way straightaway.